java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -187,6 +190,7 @@ private void execute(MethodTree tree) {
     methodTree = tree;
     constraintManager = new ConstraintManager();
     workList = new LinkedList<>();
+    endOfExecutionPath = new HashSet<>();
```

Comment:
Ok, I'll agree on that, we don't need ordered yields, nor ordered endOfExecutionpath. So let's keep it that way.